src/components/canvas/OverviewCanvas.ts

```diff
@@ -1,8 +1,10 @@
 import AbstractCanvas from '@/components/canvas/AbstractCanvas';
 import { Editor } from '@baklavajs/core';
+import Model from '@/nodes/overview/Model';
+import { Overview } from '@/nodes/model/Types';
 
 export default class OverviewCanvas extends AbstractCanvas {
   registerNodes(editor: Editor): void {
-    console.log('No nodes registered');
+    editor.registerNodeType(Overview.ModelNode, Model as any);
```

Comment:
Don't think needs to be `any`

Reply:
It does, the fact that it has a variable constructor messes with the Baklava type checking.

Reply:
I can't do 'as (NodeConstructor)' or whatever the proper type is, because it's not exposed by the Baklava packages so I would have to import from 'dist' etc.